src/Security/Security.php

```diff
@@ -138,4 +140,20 @@ public static function getPermissionsSQLByUserId(int $owner_id, bool $accessible
 		}
 		return $sql;
 	}
+
+	public static function getRegisterPolicy(): int
```

Comment:
Since the registration policy constants are defined in the `Module\Register` class, I believe this method should be moved over there as well. Or the constants be moved in the `Security` class, but it doesn't feel right to me. I'll approve either way though.

Reply:
I wasn't totally sure about the right place either.

Reply:
Like I said, it doesn't really matter as long as both the constants and this method are in the same class. Since we use these constants in the config files, it'd be less disruptive to move this method in the Register module.

Reply:
Currently the register class hasn't a `DI` implementation, also moving this would mean to possibly enhance a handful of `initialize` calls as well. Since I was slightly lazy, I wanted to avoid this 😁

Reply:
I didn't expect you to make it a class method, you can move it as a static method because it doesn't need all the module dependencies. If you don't want to add more `DI` calls to the `Register` module class, you can make the `Module\Register::getPolicy` (after move/rename) method accept a `IManageConfigValues` parameter.

Reply:
Isn't it unwanted to have static functions inside of classes where all the other ones aren't? if this is okay, I can do it that way. I just wanted to avoid too many changes, since I had to change a lot of places there.

Reply:
No, it's fine because the method is outside of the regular lifecycle of Module objects of processing a HTTP request into a HTTP response. As a result it has few dependencies (except `User` but we aren't fixing this any time soon) and it isn't a problem for it to be static.